astropy/io/fits/tests/test_connect.py

```diff
@@ -329,3 +329,13 @@ def test_unit_warnings_read_write(tmpdir):
     with catch_warnings() as l:
         Table.read(filename, hdu=1)
     assert len(l) == 0
+
+
+def test_convert_comment_convention(tmpdir):
+    """
+    Regression test for https://github.com/astropy/astropy/issues/6079
+    """
+    filename = DATA + '/table_with_comment.fits'
```

Comment:
There are other files with comments in the table header (stddata.fits, memtest.fits).
Also better to use `os.path.join` instead of string concatenation.